README.md

```diff
@@ -68,3 +68,30 @@ This command is used to build a Docker image from a Dockerfile. Here's what each
 - `.`: This specifies the build context, which is the location of the Dockerfile and any files that are being copied into the Docker image. In this case, it's the current directory.
 
 So, this command builds a Docker image for Linux systems with AMD64 architecture, tags it with the name `qsofte` and the version `v0.10`, and uses the current directory as the build context.
+
+## Setup Local Dev Environment
+
```

Comment:
@ashutoshqsofte - Add bare minimum software to install here. 

Eg . 

Install an IDE - “We use Vscode”
Install nodejs
Install git bash